packages/store/slices/settings.ts

```diff
@@ -20,6 +21,7 @@ const initialState: Settings = {
   isAdenaConnected: false,
   alreadyVisited: false,
   areTestnetsEnabled: false,
+  sideBarExpanded: true,
```

Comment:
wouldn't this have a bad behavior on mobile? (burger menu is open the first time you start the app)

Reply:
I tried on my phone, but that is not the case.
The sidebar is not expanded on load.
Is following the viewport size in here:

```
  useEffect(() => {
    setIsSidebarExpanded(
      windowWidth >= MOBILE_WIDTH ? settingIsSidebarExpanded : false
    );
  }, [settingIsSidebarExpanded, windowWidth]);
  ```